omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js

```diff
@@ -663,7 +663,7 @@ ConfirmationSectionModel.prototype.select = function() {
                     // question title is label, all fields on single line
                     else {
                         summaryDiv.append("<p><span class='title'>" + question.title().text() + ": </span>"
-                            + (question.valueAsText && !/^\s*$/.test(question.valueAsText) ? question.valueAsText : "--") + "</p>");
+                            + (question.valueAsText && (!/^\s*$/.test(question.valueAsText) ? question.valueAsText : "--")) + "</p>");
```

Comment:
What's the reasoning behind the change in parenthesis here? The ? expression is meant to return a string (either the question text or --) but now you are you are &&ing that with question.valueAsText.  Or am I missing something?

Reply:
Reading it over again, I see that I had put the parenthesis in the wrong place. The purpose of these is to improve readability as it took me a few moments to process this line. It can be removed if you would like.